src/pages/game/game-won-popup/GameWonPopup.tsx

```diff
@@ -0,0 +1,46 @@
+import { Typography, Modal } from '@mui/material';
+import { EndGameModalContainer } from './GameWonPopup.styled';
+import Confetti from 'react-confetti';
+import { useAppSelector, useWindowSize } from '../../../app/hooks';
+import { selectArticle } from '../article/articleSlice';
+
+type PropsType = {
+    onClose: () => void;
+};
+
+const GameWonPopup = ({ onClose }: PropsType) => {
+    const { width, height } = useWindowSize();
+    const { isGameWon } = useAppSelector(selectArticle);
+
+    const confettiColors = [
```

Comment:
as it is constant please capitalise variable name

Reply:
done